### ISSUE TITLE
SSH with custom prompt (PS1) returns "exit" prompt

### ISSUE THREAD

**Issue by wesleylether** [warpdotdev/Warp#793]
### Describe the bug

When you change the feature to custom prompt (PS1), then when you ssh in a server the prompt wil always be 'exit'.
See screenshots.

I use a custom prompt for color, git status information and have always the edit field on start of line.

This is my custom prompt:

```shell
PROMPT='${ret_status}%{$fg[white]%}%n%{$fg[magenta]%}@%{$fg[cyan]%}%M%{$fg[white]%}: %{$fg[magenta]%}%~%{$reset_color%} $(git_prompt_info)
%{$fg[red]%}%{$reset_color%}'
```

### To Reproduce

Steps to reproduce:
1. Go to settings and go to features
2. Enable "Honor user's custom prompt (PS1) setting"
3. Open new warp window and ssh in a server
4. See prompt returning 'exit'

### Expected behavior

In the images I have a screenshot without my custom prompt, and then it shows it correctly

### Screenshots

When I have custom prompt disabled:
![default prompt](https://user-images.githubusercontent.com/7863991/157320383-82e70f83-7735-4879-b91d-c260084f0916.png)

And when I have a custom prompt enabled:
![custom prompt](https://user-images.githubusercontent.com/7863991/157320410-70b03947-964d-4666-b523-26a4ef428e0a.png)

### Operating System

MacOS

### OS Version

12.1

**Comment by vorporeal** [comment]
@wesleylether Hey Wesley, we just pushed out a new release yesterday containing a lot of SSH fixes, including some for custom prompts - are you still running into this issue (when the PS1 feature is enabled)?

**Comment by wesleylether** [comment]
@vorporeal I have updated to the latest version, and tried again. Sadly to say, but I have still the exit problem with my custom prompt.

**Comment by vorporeal** [comment]
@wesleylether That's unfortunate; would you mind sharing your .zshrc with me?  If you don't feel comfortable sharing it here, you can email it to me at david@warp.dev (and absolutely remove any sensitive information/secrets)

**Comment by wesleylether** [comment]
@vorporeal Done! Send you an email 👍🏽

**Comment by wesleylether** [comment]
@vorporeal after the updates until now the exit message is gone, but I don't see the PS1 form the server. Wanted you to know 👍🏽

**Comment by vorporeal** [comment]
Thanks for the update!  We think we figured out what's going wrong; the
prospective fix is going out in this week's release.  I'll let you know
when it's live. :)

**Comment by elviskahoro** [comment]
Fix is out! v0.2022.04.11.09.09.stable_02
